src/documentation/pages/components-wip/tabs/examples/_vertical-lumen-centurylink.vue

```diff
@@ -0,0 +1,141 @@
+<template lang="pug">
+  <ComponentExample title="Vertical" id="vertical-lumen-centurylink" additionalClasses="-bg--grey-20" padding="-p--0" :tabs="exampleTabs" :headTabs="headTabs" @chiHeadTabsChange="e => changeSelectedTab(e)">
+    .-p--3(slot="example")
+      div(:class="'chi-grid -no-gutter -bg--' + (selectedTabId === 'base' ? 'white' : 'black')")
```

Comment:
As in the previous comments, you can use the class to create a `div` and you can avoid the `-p--3`  by removing the `padding="-p--0"`

I strongly suggest you to use interpolation instead of concatenation.

```suggestion
      .chi-grid.-no-gutter(:class="`-bg--${selectedTabId === 'base' ? 'white' : 'black'}`" slot="example")
```

Check this in every example.